connexion/operations/openapi.py

```diff
@@ -210,6 +210,10 @@ def _nested_example(self, schema):
         try:
             # Recurse if schema is an array
             return [self._nested_example(schema["items"])]
+        except KeyError:
+            pass
+        try:
+            return generate_example(schema)
         except KeyError:
             raise
```

Comment:
I don't think the `generate_example` will raise a `KeyError`. I think we only need to catch `ImportError`s here since the dependency is optional?

Same for `swagger2.py`.

Reply:
Good spot. I've reworked that code to be a bit nicer.